examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,235 @@
+kuma-injector-9c96cddc8-745r7         1/1     Running   0          70s
+```
+
+In the following steps, we will be using the pod name of the `kuma-control-plane-*************` pod. Please replace any `{KUMA_CP_POD_NAME}` with your pod name.
+
+6. Navigate into the directory where all the kuma-demo YAML files are:
+
+```
+cd kuma/examples/kubernetes/kuma-demo/
+```
+
+1. Deploy Kuma's sample marketplace application
+
+```
+$ kubectl apply -f kuma-demo-aio.yaml
+namespace/kuma-demo created
+serviceaccount/elasticsearch created
+service/elasticsearch created
+replicationcontroller/es created
+deployment.apps/redis-master created
+service/redis-master created
+service/kuma-demo-api created
+deployment.apps/kuma-demo-app created
+```
+
+This will deploy our demo marketplace application split across 3 pods. The first pod is an Elasticsearch service that stores all the items in our marketplace. The second pod is a Redis service that stores reviews for each item. The last pod is our Node/Vue application that allows you to visually query the Elastic and Redis endpoints.
+
+Check the pods are up and running by checking the `kuma-demo` namespace
+
+```
+kubectl get pods -n kuma-demo
+NAME                             READY   STATUS    RESTARTS   AGE
+es-pkm29                         2/2     Running   0          7m23s
+kuma-demo-app-5b8674794f-7r2sf   3/3     Running   0          7m23s
+redis-master-6b88967745-8ct5c    2/2     Running   0          7m23s
+```
+
+In the following steps, we will be using the pod name of the `kuma-demo-app-*************` pod. Please replace any `{KUMA_DEMO_APP_POD_NAME}` with your pod name.
+
+8. Deploy the logstash service
+
+```
+$ kubectl apply -f kuma-demo-log.yaml
+namespace/logging created
+service/logstash created
+configmap/logstash-config created
+deployment.apps/logstash created
+```
+
+9. Port-forward the sample application to access the front-end UI at http://localhost:3001
+
+<pre><code>$ kubectl port-forward <b>{KUMA_DEMO_APP_POD_NAME}</b> -n kuma-demo 8080 3001
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+Forwarding from 127.0.0.1:3001 -> 3001
+Forwarding from [::1]:3001 -> 3001
+</code></pre>
+
+Now you can access the application through your web browser at http://localhost:3001.
+
+The items on the front page are pulled from the Elasticsearch service. While the reviews for each item sit within the Redis service. You can query for individual items and look at their reviews. Happy shopping!
+
+10. Now we will port forward the kuma-control-plane so we can access it with `kumactl`
+
+<pre><code>$ kubectl -n kuma-system port-forward <b>{KUMA_CP_POD_NAME}</b> 5681
+Forwarding from 127.0.0.1:5681 -> 5681
+Forwarding from [::1]:5681 -> 5681
+</code></pre>
+
+Please refer to step 5 to copy the correct `{KUMA_CP_POD_NAME}`.
+
+11. Now configure `kumactl` to point towards the control plane address
+
+```
+$ kumactl config control-planes add --name=kuma-app --address=http://localhost:5681
```

Comment:
I think, `kuma-app` is a very confusing name for Control Plane.

`k8s` or `minikube` seem to be a better fit

Reply:
updated name of control plane